cmd/cluster/org_id.go

```diff
@@ -0,0 +1,59 @@
+package cluster
+
+import (
+	"encoding/json"
+	"fmt"
+	ctlutil "github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+type OrgId struct {
```

Comment:
You're using this so that adding flags in the future will just be stored in this struct?

Reply:
Correct, plus the consistency of having the "Run" commands of our commands tied to a struct for others to use.